upgradepolicy/docs/UpgradePolicyV2API.md

```diff
@@ -0,0 +1,473 @@
+# UpgradePolicyV2API
+
+All URIs are relative to *http://localhost/api/upgrade_policy*
```

Comment:
This should be csp.infoblox.com

In the generator, the swagger.json might be missing the following line, can you please manually add it? 
  "host": "csp.infoblox.com"

Reply:
Done